### DISCUSSION TITLE
Extensions not being updated since months?

### DISCUSSION THREAD

**Discussion by ntimo** [coder/code-server#2151] [General] [answered]
Hello,
I recently installed the extension https://github.com/gioboa/jira-plugin. After viewing the Github releases I noticed that the latest release is 0.22.1 but my code server is using 0.19.0 (release Dec 16, 2019) after that version 0.20.0 was release on Apr 25 so it seems like extensions are not being updated for months? Thats why I asked my self how often extensions are updated? Because if I understand it correctly you have to build them yourself since you host your own extension marketplace. 

The official VS Code marketplace already serves the updated version.

Thanks 👍

**Comment by bpmct** [comment, marked as answer]
Just created an issue #2153 :)

Here's some info about why we have to build them ourselves and maintain our own marketplace: https://github.com/cdr/code-server/blob/master/doc/FAQ.md#whats-the-deal-with-extensions
